src/PIL/_util.py

```diff
@@ -2,13 +2,20 @@
 import sys
 
 py3 = sys.version_info.major >= 3
+py36 = py3 and sys.version_info.minor >= 6
```

Comment:
Same comment here. 

Would it be better to define it once here and import it elsewhere?